gpjax/natural_gradients.py

```diff
@@ -0,0 +1,288 @@
+import typing as tp
+from copy import deepcopy
+
+import jax
+import jax.numpy as jnp
+import jax.random as jr
+import jax.scipy as jsp
+import optax as ox
+from jax import value_and_grad
+from jaxtyping import f64
+
+from .abstractions import InferenceState, get_batch, progress_bar_scan
+from .config import get_defaults
+from .gps import AbstractPosterior
+from .parameters import (
+    build_identity,
+    build_trainables_false,
+    build_trainables_true,
+    trainable_params,
+    transform,
+)
+from .types import Dataset, PRNGKeyType
+from .utils import I
+from .variational_families import (
+    AbstractVariationalFamily,
+    ExpectationVariationalGaussian,
+    NaturalVariationalGaussian,
+)
+from .variational_inference import StochasticVI
+
+DEFAULT_JITTER = get_defaults()["jitter"]
+
+
+def natural_to_expectation(
+    natural_moments: dict, jitter: float = DEFAULT_JITTER
```

Comment:
Import `Dict` from typing in your imports and then use `Dict` as the type.

Reply:
Absolutely. This is something that is inconsistent across the package. Will address this in the next commit.